### PR TITLE
Fix flickering animation

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

**Comment by Joozty** [comment]
Looks like Safari is playing a game with me. Firefox and Chrome work like a charm. :(

**Comment by Joozty** [comment]
@viktor-yakubiv what do you think about this workaround? I am not sure if there is a way to fix that right padding in safari.

**Comment by Joozty** [comment]
Negative delay skips the first Xms of animation. The animation is not that smooth as before but it's not bad either.

File: src/elements/forms/styles.css [hunk 1]
```diff
@@ -34,7 +34,7 @@
   line-height: 1;
   color: var(--form-label-color);
   border-radius: var(--form-control-corner-radius, 2px);
-  transition: 150ms;
+  transition: all 150ms ease -50ms;
```
Comment (viktor-yakubiv) [inline src/elements/forms/styles.css:37]: Ease is the default value, there was no need to add it.

**Pull request merged**